config/locales/server.en.yml

```diff
@@ -10,8 +10,21 @@ en:
     unassign_on_group_archive: "When a message is archived by a group, unassign message (reassign if moved back to inbox)"
     unassign_on_close: "When a topic is closed unassign topic"
     assign_locks_flags: "When a topic is assigned to a staff member, its flags can only be handled by that person"
+    assign_mailer_enabled: "When enabled, the assigned user will receive a notification email on each assignment"
+    assign_mailer_disabled_for_muted_topics: "When enabled, the assigned user will not receive emails on muted topics"
```

Comment:
This is overdoing it I would remove this option, if email_on_assignment is enabled then it should never assign on muted topics you should not be allowed to override it.

Reply:
Thanks for reviewing! Actually user always receive emails on muted topics if 'assign_mailer_disabled_for_muted_topics' disabled or absent. So it is not overrided by any other site setting.

Reply:
Sure I understand, I just feel it is an option we don't need to provide.... I would like to delete the setting "assign_mailer_disabled_for_muted_topics"